video/filter/vf_d3d11vpp.c

```diff
@@ -506,6 +780,8 @@ const struct mp_user_filter_entry vf_d3d11vpp = {
             .deint_enabled = true,
             .mode = D3D11_VIDEO_PROCESSOR_PROCESSOR_CAPS_DEINTERLACE_BOB,
             .field_parity = MP_FIELD_PARITY_AUTO,
+            .scale = SUPER_RESOLUTION_OFF,
+            .scale_target = SUPER_RESOLUTION_AUTO,
```

Comment:
>Current "scale" should be renamed to scale_method or similar. Although since currently none of the vendors offer selection, this can be made as a boolean value to enable high quality scalers.

You mean none of the currently supported vendors? If this is expanded to AMD, they do support different methods with AMF_HQ_SCALER_ALGORITHM.

Reply:
At current stage, there is no need for such option. `scaling_algo`can be added in the future when it will be useful.